biodatasets/mediqa_nli/mediqa_nli.py

```diff
@@ -144,9 +142,10 @@ def _split_generators(self, dl_manager) -> List[datasets.SplitGenerator]:
         if self.config.data_dir is None:
             raise ValueError("This is a local dataset. Please pass the data_dir kwarg to load_dataset.")
         else:
-            data_dir = os.path.join(
-                dl_manager.extract(self.config.data_dir), "mednli-for-shared-task-at-acl-bionlp-2019-1.0.1"
+            _ = dl_manager.extract(
+                os.path.join(self.config.data_dir, "mednli-for-shared-task-at-acl-bionlp-2019-1.0.1.zip")
             )
+            data_dir = os.path.join(self.config.data_dir, "mednli-for-shared-task-at-acl-bionlp-2019-1.0.1")
```

Comment:
I think this will only work on your machine b/c you already unzipped it .. I'll make a quick edit, but this looks ready to merge!  (just fyi, the extract function will return the path to the extracted data so we need to use that when defining `data_dir`)